specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerConnectivityConfiguration.json

```diff
@@ -0,0 +1,414 @@
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/connectivityConfigurations/{configurationName}": {
+      "parameters": [
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkManagerNameParameter"
+        },
+        {
+          "$ref": "#/parameters/ConnectivityConfigurationNameParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "ConnectivityConfigurations"
+        ],
+        "operationId": "ConnectivityConfigurations_Get",
+        "description": "The Get ConnectivityConfigurations operation gets a Managed Network Connectivity Configuration resource, specified by the  resource group, Managed Network name, and connectivity Configuration name",
+        "responses": {
+          "200": {
+            "description": "Get successful. The operation returns the specified Managed Network Connectivity Configuration resource.",
+            "schema": {
+              "$ref": "#/definitions/ConnectivityConfiguration"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConnectivityConfigurationsGet": {
+            "$ref": "./examples/NetworkManagerConnectivityConfigurationGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ConnectivityConfigurations"
+        ],
+        "operationId": "ConnectivityConfigurations_CreateOrUpdate",
+        "description": "The Put ConnectivityConfigurations operation creates/updates a new Managed Network Peering Configuration",
+        "parameters": [
+          {
+            "name": "connectivityConfiguration",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConnectivityConfiguration"
+            },
+            "description": "Parameters supplied to create/update a Managed Network Peering Configuration"
+          }
+        ],
+        "responses": {
```

Comment:
As PUT is a long running operation, add response 202 to the list. Refer https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers

Reply:
Our PUT operation is not async operation, and reponse will be sent after the whole process completed. And since we only do 
simple steps like create and store it to cosmosdb, so PUT operation isn't running long. In this situation, should we still need to add 202? And should we have to change all put operation to aysnc?

Reply:
If your PUT is not async, please remove x-ms-long-running-operation fields.